build/lin.sh

```diff
@@ -158,6 +178,11 @@ make install
 mkdir ${DEPS}/ffi
 $CURL https://github.com/libffi/libffi/releases/download/v${VERSION_FFI}/libffi-${VERSION_FFI}.tar.gz | tar xzC ${DEPS}/ffi --strip-components=1
 cd ${DEPS}/ffi
+if [ $DARWIN_ARM = true ]; then
```

Comment:
Variable needs to be quoted before comparison.
```suggestion
if [ "$DARWIN_ARM" = true ]; then
```